rfcs/20230323-add-split-graph-op/README.md

```diff
@@ -0,0 +1,152 @@
+# Add Split Operation in Graph API
+
+## Motivation
+
+`Split` operation splits an input tensor into pieces of the specific length
+along some axis, which is opposite to the operation of `concat`. `Split`
+operation is supported by different deep learning frameworks and toolkit, eg.
+ PyTorch [1], TensorFlow [2], OpenVINO [3], and ONNX [4]. The `split`+`concat` 
+ pattern is more common in most models and takes up a large amount of computing 
+ time. However, since the graph API does not support `Split` op, possible 
+ optimization is limited. So we propose to add the `Split` op into graph API.
+
+
+## Proposal
```

Comment:
Before proposing options for graph API, I feel it would be better to add a section describing how the op and functionality are implemented in mainstream frameworks and toolkits.

Reply:
@TaoLv  Done for that, please help to check